.gitignore

```diff
@@ -5,4 +5,5 @@
 
 # Files
 .DS_Store
-composer.lock
\ No newline at end of file
+composer.lock
+*.bat
```

Comment:
Can you please add trailing newline here? Don't know why there wasn't one previously.

Reply:
Please how can I do the changes that you have requested? Do I need to cancel this PR first?

Reply:
Since your pull request is to pull FSElias:master into netsuitephp:master, all you need to do is add a new commit to FSElias:master. Most editors have a setting that will detect and automatically correct any missing trailing newline in a file. The reasons to do so are sort of edge case things, but it's considered enough of a standard that github points it out in differences and so I try to make sure it is fixed anywhere it pops up in code I manage.

Reply:
Done